examples/http-retry/http_retry.bal

```diff
@@ -3,26 +3,20 @@ import ballerina/io;
 
 public function main() returns error? {
     http:Client httpClient = check new ("localhost:9090",
-        // Retry configuration options.
-        retryConfig = {
+    retryConfig = {
```

Comment:
This formatting is a bit odd right? How about adding the `retryConfig = {` to the previous line?
```bal
   http:Client httpClient = check new ("localhost:9090", retryConfig = {
```
Or we could keep the original formatting
(This formatting issue is there for other BBEs as well)